goldens/cdk/collections/index.api.md

```diff
@@ -1,4 +1,4 @@
-## API Report File for "components-srcs"
+## API Report File for "@angular/cdk_collections"
```

Comment:
Nit: In an ideal world we would have the package string listing here correctly end up with the deeper entry point `@angular/cdk/collections`

Reply:
Agreed, that's a fix we'd need to make in `api_golden`